lib/services/feature_flag_service.dart

```diff
@@ -83,6 +84,15 @@ class FeatureFlagService {
     }
   }
 
+  bool enableSearchFeature() {
+    String email = Configuration.instance.getEmail();
+    if (email.endsWith("@ente.io") || email == 'ashilkn99+ente77@gmail.com') {
```

Comment:
email id can be null during sign up. Because we render the home widget with multiple pages, do verify that it's not breaking anything.

Reply:
Verified. Whenever enableSearchFeature() is called, email id is always != null.